tensorboard/components/tf_card_heading/tf-card-heading-style.html

```diff
@@ -0,0 +1,47 @@
+<!--
+@license
+Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!--
+  Shared stylesheet for figure captions.
+-->
+<dom-module id="tf-card-heading-style">
+  <template>
+    <style>
+
+      /** Horizontal line of labels. */
+      .row {
```

Comment:
This CSS is included within various modules, and it affects all elements within the modules. Could we prefix these declarations with `tf-card-heading`? ie,

```css
tf-card-heading .row {
   ...
}
```
etc. 

to prevent the declarations from affecting unrelated elements (say also with the class `row` or `label`).

Reply:
Done.